docker/docker-compose.yaml

```diff
@@ -1,4 +1,4 @@
-version: '3.7'
+version: '3.11'
```

Comment:
hm is this just a search replace gone wrong? this setting is for docker and not the python version